include/crm/msg_xml.h

```diff
@@ -280,6 +280,7 @@ extern "C" {
 #define PCMK_XA_LAST_RC_CHANGE              "last-rc-change"
 #define PCMK_XA_LAST_UPDATED                "last_updated"
 #define PCMK_XA_LOCKED_TO                   "locked_to"
+#define PCMK_XA_LOCKED_TO_HYPHEN            "locked-to"
```

Comment:
That's annoying :/ This sort of thing shouldn't happen going forward, thanks to everything being in one place.

Reply:
agreed

Reply:
We also have some that aren't technically conflicts but still suck, like `PCMK_XA_TARGET_ROLE` (`"target_role"`) vs. `PCMK_META_TARGET_ROLE` (`"target-role"`).